### PR TITLE
Fibonacci homework Lukasz K/Nanautzin

### PR DESCRIPTION
**Pull request by lukasz-kukulka**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: fibonacci
🏅 Granted 6 XP for @lukasz-kukulka

**Comment by ziobron** [comment]
⏱ 3 XP extra for @lukasz-kukulka for on-time delivery

File: homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    if (sequence < 1){
+        return 0;
+    } else if (sequence == 2 || sequence == 1) { 
+        return 1;
+    }
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:18]: If you return variable here, you don't need to use `else if`

**Review by nauka-programowania-MA — APPROVED** [review]
Perfect! I really enjoy your solutions!

**Pull request closed without merging**